src/containers/ImportResources/ImportResources.js

```diff
@@ -365,7 +365,7 @@ export class ImportResources extends Component {
                 helperText={intl.formatMessage({
                   id: 'dashboard.importResources.serviceAccount.helperText',
                   defaultMessage:
-                    'The ServiceAccount that the PipelineRun will run under (from the namespace above)'
+                    "The ServiceAccount that the PipelineRun applying resources will run unde (from the namespace above). Ensure the ServiceAccount (and if it's a default one used when no ServiceAccount is specified), has permissions for creating PipelineRuns, PipelineResources, and any Tekton resources in the Git repository."
```

Comment:
typo: 'under'

maybe similar rewording to the Create PipelineRun text for the ServiceAccount permissions piece